tools/pipelines/test-coverage.yml

```diff
@@ -0,0 +1,143 @@
+# Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+# Licensed under the MIT License.
+
+# test-coverage pipeline (for the purpose of detecting flaky tests)
```

Comment:
Rename to "test-reliability" for better accuracy. We don't actually care about measuring coverage here.